modules/mobile_session.lua

```diff
@@ -86,14 +86,24 @@ function mt.__index:StartHeartbeat()
   self.mobile_session_impl:StartHeartbeat()
 end
 
+function mt.__index:SendHeartbeatAck()
+  self.mobile_session_impl:SendHeartbeatAck()
+end
+
 function mt.__index:SetHeartbeatTimeout(timeout)
   self.mobile_session_impl:SetHeartbeatTimeout(timeout)
 end
 
 --! @brief Start service 7 and heartBeat
 --! @return return expectation for expectation for StartService Ack
-function mt.__index:StartRPC()
-  return self.mobile_session_impl:StartRPC()
+function mt.__index:StartRPC(custom_hb_processor)
+  local hb_processor = custom_hb_processor
```

Comment:
@VVeremjova Please consider an option of implementation without an additional variable `hb_processor`
For example:
```
function mt.__index:StartRPC(custom_hb_processor)
  custom_hb_processor = custom_hb_processor  
       or function (_,_) 
              self.mobile_session_impl:AddHeartbeatExpectation()
   	    end
  return self.mobile_session_impl:StartRPC():Do(custom_hb_processor)
end
```